Assets/RGLUnityPlugin/Scripts/SceneManager.cs

```diff
@@ -75,6 +75,7 @@ public enum MeshSource
         public static ITimeSource TimeSource { get; set; } = new UnityTimeSource();
 
         private int lastUpdateFrame = -1;
+        private int lastPhysicsCycle = -1;
```

Comment:
I'd suggest sticking with Unity-specific names.
```suggestion
        private int lastFixedUpdateFrame = -1;
```